### DISCUSSION TITLE
C++ 基础知识小讲堂 (4)—— 从零开始的 vector 实现 (1) – ChloroBlog

### DISCUSSION THREAD

**Discussion by Chlorie** [Chlorie/ChloroBlog#27] [Announcements]
https://chlorie.github.io/ChloroBlog/posts/2020-04-29/0-cpp-basics-4.html 

 能鸽善鹉者的领域

**Comment by c0nstexpr** [comment]
这语言我根本学不下去.jpg

**Comment by wo315** [comment]
非常有意思。不过部分内容，尤其是第五部分，我还得消化吸收一下。

非常棒！
